ManatiCR/retopais
# RP-55: Judge view and permissions.

**To do:**
- [x] Add new roles.
- [x] Add new permissions.
- [x] Add new view to see proposals that have been approved.
- [x] Config the views with the correct permissions.
- [x] Configure the roles with the correct permissions.

**Steps to test:**
- Check that there are two new roles: content admin (the two big bosses), and judge (the ones who'll see the pre-approved proposals).
- Add proposals.
- As a content admin you should see on the management menu a 'Ver propuestas' link where you can pre-approve a proposal.
- As a judge you should see on the management menu a 'Ver propuestas finales' link, where you can see only the pre-approved proposals.
